src/devtoolsPanel.ts

```diff
@@ -420,19 +422,20 @@ export class DevToolsPanel {
                 ">
             </head>
             <body>
-                <iframe id="devtools-frame" frameBorder="0" src="${cdnBaseUrl}?experiments=true&theme=${theme}"></iframe>
+                <iframe id="devtools-frame" frameBorder="0" src="${cdnBaseUrl}?experiments=true&theme=${theme}&headless=${this.isHeadless}"></iframe>
             </body>
             </html>
             `;
     }
 
-    private setBrowserRevision(revision: string) {
-        if (revision !== '') {
+    private setBrowserRevision(msg: {revision: string, isHeadless: boolean}) {
```

Comment:
My small NIT is that the function/event name `setBrowserRevision` doesn't relate to setting and grabbing the headless state.  Since this is a temporary solution to grab the headless state, I'm more okay with leaving it in, but it is a minor change either way.

Reply:
Thats a good point. I think it makes sense to update the name of the event in response.